src/common/gas-station/index.ts

```diff
@@ -0,0 +1,50 @@
+import { BigNumber, ethers } from "ethers";
+import fetch from "node-fetch";
+
+export type GasStationFunction = (gasStationUrl: string) => () => Promise<GasStationFeeData | undefined>;
+export type GasStationFeeData = { maxPriorityFeePerGas: BigNumber | null; maxFeePerGas: BigNumber | null };
+
+export const gasStation: GasStationFunction =
+  (gasStationUrl: string) => async (): Promise<GasStationFeeData | undefined> => {
+    try {
+      if (!gasStationUrl) return undefined;
+      const res = await fetch(gasStationUrl);
+      const data = await res.json();
+      return {
+        maxPriorityFeePerGas: safeParseUnits(data.standard.maxPriorityFee.toString(), 9),
+        maxFeePerGas: safeParseUnits(data.standard.maxFee.toString(), 9),
+      };
+    } catch (e) {
+      throw new Error("Failed to fetch gas station");
+    }
+  };
+
+const safeParseUnits = (_value: number | string, decimals: number): BigNumber => {
```

Comment:
I saw in line 39 which states to `Pad the fraction to 9 decimal places`, would it be better to set the param `decimals` to a default value of `9`?

Reply:
I was thinking would it be better if we default the decimals in `gasStation` function instead? Like this: https://github.com/Open-Attestation/open-attestation-cli/pull/292/commits/a16021bd30dd9df5c1393935d1eefc340ab5210e#diff-609663441672651e69555c0a1ae9ed0a7ec39717d6a59f38c05484b904018362R8-R9

Then the `safeParseUnits` (which I modified from ethersjs btw) can remain generic.